TextToSpeech/TextToSpeechImplementation.cpp

```diff
@@ -62,6 +65,17 @@ namespace Plugin {
         JsonObject config;
         config.FromString(service->ConfigLine());
 
+        InputValidation::Instance().addValidator("ttsendpoint", ExpectedValues<std::string>("^https?://[a-zA-Z0-9]+.*"));
+        InputValidation::Instance().addValidator("ttsendpointsecured", ExpectedValues<std::string>("^https://[a-zA-Z0-9]+.*"));
+        InputValidation::Instance().addValidator("double_str", ExpectedValues<std::string>("^-?[0-9]+(\\.[0-9]+)?"));
+        InputValidation::Instance().addValidator("speechid", ExpectedValues<uint32_t>(1, UINT32_MAX));
+        InputValidation::Instance().addValidator("text", ExpectedValues<std::string>("^[a-zA-Z0-9]+"));
+        InputValidation::Instance().addValidator("speechrate", ExpectedValues<std::string>({"slow", "medium", "fast", "faster", "fastest"}));
+        InputValidation::Instance().addValidator("rate", (std::function<bool(const uint8_t &)>)[](const uint8_t &val) -> bool { return (val >= 0 && val <=100); });
```

Comment:
'rate', 'volume' & 'setPrimaryVolDuck' values can be set as 'ExpectedValues<uint8_t>(0, 100)'